JournalMonitor/JournalMonitor.cs

```diff
@@ -217,7 +215,7 @@ public static List<Event> ParseJournalEntry(string line, bool fromLogLoad = fals
                                         factions = getFactions(factionsVal);
                                     }
 
-                                    events.Add(new JumpedEvent(timestamp, systemName, systemAddress, x, y, z, starName, distance, fuelUsed, fuelRemaining, boostUsed, allegiance, faction, factionState, economy, economy2, government, security, population, factions) { raw = line, fromLoad = fromLogLoad });
+                                    events.Add(new JumpedEvent(timestamp, systemName, systemAddress, x, y, z, starName, distance, fuelUsed, fuelRemaining, boostUsed, allegiance, faction.name, faction.FactionState, economy, economy2, government, security, population, factions) { raw = line, fromLoad = fromLogLoad });
```

Comment:
Consider passing faction as an object? We're reconstituting the object at EDDI.cs line 1280. We can simplify if we build the object here and simply pass the object.